contrib/googlejavaformat/src/python/pants/contrib/googlejavaformat/googlejavaformat.py

```diff
@@ -0,0 +1,76 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from abc import abstractproperty
+
+from pants.backend.jvm.tasks.scala_rewrite_base import ScalaRewriteBase
+from pants.base.exceptions import TaskError
+from pants.java.jar.jar_dependency import JarDependency
+from pants.task.fmt_task_mixin import FmtTaskMixin
+from pants.task.lint_task_mixin import LintTaskMixin
+
+
+class GoogleJavaFormatBase(ScalaRewriteBase):
+
+  _MAIN = 'com.google.googlejavaformat.java.Main'
```

Comment:
How about inlining this? I _think_ its only needed here and in one place and not a magic `_OVERRIDE_ME` like the constants below.